src/lib/svg_text_utils.js

```diff
@@ -264,19 +210,14 @@ var ENTITY_TO_UNICODE = Object.keys(stringMappings.entityToUnicode).map(function
     };
 });
 
-var UNICODE_TO_ENTITY = Object.keys(stringMappings.unicodeToEntity).map(function(k) {
```

Comment:
Oh I see, right, this is useless now as we're building the text nodes directly instead of using `window.DomParser`. Nice!